storybook/src/video/videoProperties.js

```diff
@@ -0,0 +1,57 @@
+export default {
+  'vid:transcodedVideos': [
+    {
+      name: 'WebM 480p',
+      content: {
+        name: 'sample.webm',
+        'mime-type': 'video/webm',
+        encoding: null,
+        digestAlgorithm: 'MD5',
+        digest: '046356f7764cbd67acfc4365eb106e06',
+        length: '401187',
+        data: 'videoURL',
```

Comment:
We could have a real url here, pointing a small video. Otherwise the download button will be broken.

Reply:
finding a url with video (.mp4) is pretty hard, can we point it to nuxeo website?

Reply:
We should, as much as possible, not rely in external resources in this context.

Reply:
We should use a local resource. We can copy the `sample.mp4` file which we already use on the legacy demos.

Reply:
i tried, but if i add the relative path it doesn;t work